src/fmap.c

```diff
@@ -0,0 +1,70 @@
+// Flashmap support.
+//
+// Copyright (C) 2019 3mdeb
+//
+// This file may be distributed under the terms of the GNU LGPLv3 license.
+
+#include "malloc.h" // malloc
+#include "string.h" // memcpy
+#include "output.h" // dprintf
+#include "fmap.h"   // find_fmap_directory
+
+static void* fmap_entry = NULL;
+
+#define ROM_BEGIN       ((void *)0xFF800000)
+#define ROM_END         ((void *)0xFFFFFFFF)
+
+void find_fmap_directory(void)
+{
+    void *offset = ROM_BEGIN;
+
+    while (offset <= ROM_END) {
```

Comment:
@piotr-kleins during the development I did not know yet about the fmap_offset thing in coreboot tables. Check whether we can access fmap offset in lib_sysinfo.fmap_offset